telco-hub/configuration/reference-crs/optional/backup-recovery/backupSchedule.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: BackupSchedule
+metadata:
+  name: schedule-drtest
+  namespace: open-cluster-management-backup
+spec:
+  veleroSchedule: "0 */2 * * *" #backup set every 5 mns
```

Comment:
Isn't this every 30 minutes?

Reply:
my mistake , it is every 2 hours indeed.